conans/migrations.py

```diff
@@ -32,6 +32,8 @@ def _make_migrations(self, old_version):
     def _update_version_file(self):
         try:
             save(self.file_version_path, str(self.current_version))
+        except OSError as error:
```

Comment:
Better just capture ``except Exception as error`` and include the ``str(error)`` message in the "Can't write version file in...", as it would be more complete and informative

Reply:
Yes, as utils.file.save raises an informative message, my concern was about duplicated message. But in terms of code, it's too generic here, we can't deduce the problem without checking save() first.

Reply:
Done.